hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -126,31 +120,6 @@
 
   private DUFactory.Conf duConf;
```

Comment:
Since this isn't getting initialised now, we can remove this and its usages in the `setBalancingInterval` method.